src/tests/system/tests/test_ipa.py

```diff
@@ -390,3 +390,122 @@ def test_ipa__check_gssapi_authentication_indicator(client: Client, ipa: IPA):
+
+    ipa.user("user-1").add().iduseroverride().add_override(
+        "testview1",
+        uid=1234567,
+        gid=7654321,
+        gecos="This is the ID user override",
+        home_directory="/home/newhomedir",
+        shell="/bin/newloginshell",
+        login="newuser",
+        certificate=certificate_content,
+    )
+    ipa.group("group-1").add().idgroupoverride().add_override(
+        idview_name="testview1", group_name="newgroup", gid=88888, description="This is ID group override"
+    )
+
+    client.sssd.restart()
+
+    result = ipa.user("user-1").iduseroverride().show_override("newview1")
+
+    for user in ["user-1", "newuser"]:
+        userlookup = client.tools.getent.passwd(user)
+        assert userlookup.uid == 1234567, "overriden uid not found"
+        assert userlookup.name == "newuser", "overriden login name not found"
+        assert userlookup.gid == 7654321, "overriden gid not found"
+        assert userlookup.home == "/home/newhomedir", "overriden home directory not found"
+        assert userlookup.shell == "/bin/newloginshell", "overriden shell not found"
+
+    for group in ["group-1", "newgroup"]:
+        grouplookup = client.tools.getent.group(group)
+        assert grouplookup.name == "newgroup", "overriden group name not found"
+        assert grouplookup.gid == 88888, "overriden gid not found"
+
+    assert certificate_content in result.get("usercertificate", [""])[0], "Certificate content mismatch!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__idview_non_override_attribute(client: Client, ipa: IPA):
+    """
+    :title: Verify an IPA ID view can not override non override attributes on the client
+    :description:
+        To ensure that an IPA ID view can not override non override attributes on the client system.
+    :setup:
+        1. Client is configured to use ID view
+        2. Add a user with non override attributes
+        4. Restart SSSD on client
+    :steps:
+        1. Check IPA user's non override attributes should not be overridden
+    :expectedresults:
+        1. Non override attributes should not be overridden
+    :customerscenario: False
+    """
+
+    ipa.idview("testview").add(description="This is a new view")
+    ipa.idview("testview").apply(hosts=[f"{client.host.hostname}"])
+
+    ipa.user("user-1").add().iduseroverride().add_override("testview", email="email.com", fname="fname", lname="lname")
+
+    client.sssd.restart()
+
+    result = ipa.user("user-1").iduseroverride().show_override("testview")
+
+    assert not result.get("email", [None])[0], "Get non override attribute email in result"
+    assert not result.get("fname", [None])[0], "Get non override attribute fname in result"
+    assert not result.get("lname", [None])[0], "Get non override attribute lname in result"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__idview_apply_on_server(ipa: IPA):
```

Comment:
Just a note that 'Default Trust View' should apply values on the server.

Reply:
Should I add a note, or should I change the name?

Reply:
I think it's okay to leave as is.

Reply:
Resolving this conversation, this was just a FYI.